src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,49 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style-type: none;
+    padding-left: 0;
+    margin: 0;
+  }
+
+  &__item {
+    display: flex;
+    margin: 0 10px;
+    position: relative;
+
+    &:last-child {
+      margin-right: 0;
+    }
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  &__link {
+    line-height: 60px;
+    color: $font-main-color;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: $medium-font-weight;
+
+    &--active {
+      color: $button-main-color;
+
+      &:hover {
+        color: $button-main-color;
+      }
```

Comment:
you effectively disable all links to change font color by nesting :hover under --active modifier
make sure all nav links change color on hover